pytorch/src/gen/java/org/bytedeco/pytorch/AdaptiveAvgPool1dImpl.java

```diff
@@ -44,9 +44,6 @@ private native void allocate(
           @Const @ByRef AdaptiveAvgPool1dOptions options_);
     /** Pointer cast constructor. Invokes {@link Pointer#Pointer(Pointer)}. */
     public AdaptiveAvgPool1dImpl(Pointer p) { super(p); }
-    /** Downcast constructor. */
-    public AdaptiveAvgPool1dImpl(Module pointer) { super((Pointer)null); allocate(pointer); }
-    @Namespace private native @SharedPtr @Name("SHARED_PTR_NAMESPACE::dynamic_pointer_cast<torch::nn::AdaptiveAvgPool1dImpl, torch::nn::Module>") void allocate(@SharedPtr Module pointer);
```

Comment:
Why have **all** the "Downcast constructor" been removed??

Reply:
In 2.2, they removed the virtual inheritance between `Module` and its subclasses. So I removed the `upcast` info.